frontend/templates/troubleshooting/solution.tsx

```diff
@@ -0,0 +1,217 @@
+import {
+   Box,
+   Heading,
+   Stack,
+   Text,
+   Avatar,
+   Button,
+   Icon,
+   Badge,
+   Square,
+   Flex,
+} from '@chakra-ui/react';
+import {
+   faCircleCheck,
+   faEllipsis,
+   faPenToSquare,
+   faSquareArrowUp,
+} from '@fortawesome/pro-solid-svg-icons';
+import { FaIcon } from '@ifixit/icons';
+import { Section } from './hooks/useTroubleshootingProps';
+import Prerendered from './prerendered';
+
+const SolutionFooter = () => (
+   <Stack
+      justify="flex-start"
+      align="flex-start"
+      spacing="4px"
+      width="798.74px"
```

Comment:
I would round this to a whole pixel, 800px maybe?

Reply:
[e6254ce](https://github.com/iFixit/react-commerce/pull/1305/commits/e6254ce7840cc1b37f81bf901bf74bfe89d49670)